File: h2o-docs/src/booklets/v2_2015/source/deeplearning/deeplearning_gridsearch_result.py
# TO DO: Add Python version
# (Python Grid API is not complete yet)

# Example R version below
# # print out all prediction errors and run times of the models
# model_grid
# 
# # print out the auc for all of the models
# grid_models <- lapply(model_grid@model_ids, function(model_id) { model = h2o.getModel(model_id) })
# for (i in 1:length(grid_models)) {
#   print(sprintf("auc: %f", h2o.auc(grid_models[[i]])))
# }

File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_gridsearch_result.R
# print out all prediction errors and run times of the models
model_grid

# print out the Test MSE for all of the models
for (model_id in model_grid@model_ids) {
  mse <- h2o.mse(h2o.getModel(model_id), valid = TRUE)
  print(sprintf("Test set MSE: %f", mse))
}
